### ISSUE TITLE
Nui error when closing neovim window with only neo-tree opened in it

### ISSUE THREAD

**Issue by charbelnicolas** [MunifTanjim/nui.nvim#186]
When using neo-tree in neovim, if the last window open is the neo-tree window, when you close it you get the following error:

![20220820_110328](https://user-images.githubusercontent.com/14960414/185756127-4e5d6646-305d-470f-ab4e-e4aaf80af821.png)

**Comment by cseickel** [comment]
With my config, I noticed that I can no longer close neovim with a `:qa` if I have a neo-tree window open. This started after I saw this issue and ran a `PackerSync` just now. I was on a5fd005263d238d2fbd6ee335e06139645f11fa9 before the sync. This also happens with a completely default config.

If I set `close_if_last_window =  true` and close the last window, there is a message but I only see the "Press ENTER or ..." message line itself so I can't tell if it might be the same as OP.

I am on `NVIM v0.8.0-dev-nightly-6-g53c398d8f`
